stock_by_warehouse_stock/models/stock_move_line.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Vauxoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
Manifest says LGPL.

I'd say remove this to be safe.

Reply:
@CarmenMiranda Change the licence please.

Reply:
Done